docs/azure/sdk/snippets/dependency-injection/WebApplicationBuilder/Program.cs

```diff
@@ -0,0 +1,53 @@
+﻿#region snippet_WebApplicationBuilder
+using Azure.Identity;
+using Microsoft.Extensions.Azure;
```

Comment:
Do we not need a using for the Swagger APIs?

Reply:
No using needed.